src/components/header/modals/ModalAccountUnification.vue

```diff
@@ -69,12 +120,23 @@ export default defineComponent({
       window.removeEventListener('resize', onHeightChange);
     });
 
+    const updateSteps = (step: number) => {
```

Comment:
const updateSteps = (step: number): void => {

Reply:
Fixed it